eng/testing/tests.browser.targets

```diff
@@ -15,7 +15,7 @@
       - For regular library tests, it will use the symbols file from the runtime pack.
       - for AOT library tests, we use WasmNativeStrip=false, so we already have symbols
     -->
-    <WasmNativeStrip Condition="'$(WasmNativeStrip)' == ''">false</WasmNativeStrip>
+    <WasmNativeStrip Condition="'$(WasmNativeStrip)' == '' and '$(RunAOTCompilation)' == 'true'">false</WasmNativeStrip>
```

Comment:
Could you please explain why are we adding the condition. I'm just curious, thanks.

Reply:
Setting `WasmNativeStrip=false` would cause relinking, and without changing this condition we would be relinking all the library tests!

Reply:
is `RunAOTCompilation` here meaning "are we re-linking already?" 
Should `WasmBuildNative` be there instead ? (after it was fully resolved)

Reply:
No, it means use `WasmNativeStrip=false` only when for AOT, so only for the AOT jobs, and not the non-AOT library test jobs.

Reply:
Oh, got it, this is just for tests